api_client/src/lib.rs

```diff
@@ -32,6 +32,7 @@ pub enum StatusCode {
     NoContent,
     BadRequest,
     NotFound,
+    TooManyRequests,
```

Comment:
Can you split this change to its own commit? This is only for `api_client`, e.g. ch-remote for now. It took me a while to recall this, as I was confused why we need to define `StatusCode` again from our code base while it was added to the mirco-http crate already.

Reply:
Done!